src/main/scala/mesosphere/marathon/raml/GroupInfoConversion.scala

```diff
@@ -0,0 +1,29 @@
+package mesosphere.marathon
+package raml
+
+import mesosphere.marathon.core.appinfo.{GroupInfo => CoreGroupInfo}
+
+trait GroupInfoConversion extends AppConversion {
+
+  implicit val podRamlWriter: Writes[CoreGroupInfo, GroupInfo] = Writes { info =>
+    GroupInfo(
+      id = info.group.id.toString,
+      dependencies = info.group.dependencies.map(_.toString),
+      version = Some(info.group.version.toOffsetDateTime),
+      enforceRole = Some(info.group.enforceRole),
+      apps = info.maybeApps.fold(Set.empty)(apps => apps.toRaml)
```

Comment:
So, this is interesting. The API does *not* define a `AppInfo` but an `App` class. There is a custom JSON format for `AppInfo`. I would introduce a `raml.AppInfo`. Now, `core.appinfo.GroupInfo` and `core.appinfo.AppInfo` are actually API models. Should we simply replace them are not convert at all? This is how we do it for `info.maybePods` btw.